frontend/src/scenes/billing/PlanComparison.tsx

```diff
@@ -109,44 +108,9 @@ export const PlanComparison = ({
         return null
     }
     const fullyFeaturedPlan = plans[plans.length - 1]
-    const { reportBillingUpgradeClicked } = useActions(eventUsageLogic)
-    const { redirectPath, billing } = useValues(billingLogic)
+    const { billing } = useValues(billingLogic)
     const { width, ref: planComparisonRef } = useResizeObserver()
 
-    const upgradeButtons = plans?.map((plan) => {
```

Comment:
These buttons were also used on the billing page in the plan comparisons, seems like we should keep them there?

Reply:
Great point, I'll add them back.